astropy/io/ascii/cds.py

```diff
@@ -5,7 +5,8 @@
   Classes to read CDS / Vizier table format
 
 :Copyright: Smithsonian Astrophysical Observatory (2011)
-:Author: Tom Aldcroft (aldcroft@head.cfa.harvard.edu)
+:Author: Tom Aldcroft (aldcroft@head.cfa.harvard.edu), \
```

Comment:
This goes beyond the scope of this PR, but I wonder @taldcroft - do we still need these author metadata? I suppose the copyright to SAO is not changeable to "Astropy" that easily, but the author info/history is pretty well tracked by git itself.

Reply:
This code was copied from the `asciitable` package, which unfortunately does not specify a license. I think most licenses would require maintaining Copyright and Author information, but IANAL. Though I personally don't care, astropy should be strict about respecting typical license agreements when copying open source code.

Reply:
However, the person controlling the copyight is @taldcroft (or SAO) here and he could, should @taldcroft/SAO could chose to do transfer the copyright or re-license under a different license if they chose to.

Reply:
On a more practical note: @Suyog7130 included a number of lines straight from pycdsreadmy. That library is BSD licensed and thus we *can* include code from there into astropy, but my understanding is that we need to mark is as BSD licensed. So, we should (1) contact the authors and ask them to re-license under MIT licence, or (2) isolate all those lines into a separate file with a different license header or (3) remove that code and write it from scratch, using pycdsreadme as inspiration, but not copy code directly from there.

Reply:
`# Licensed under a 3-clause BSD style license - see LICENSE.rst`
This line appears at the top of `cds.py`. Doesn't this mean that the code in `cds.py` is BSD licensed?

Reply:
Oh, good. Great. I guess I got confused about who uses which license.